### ISSUE TITLE
Please increase emphasis on technical work of Web accessibility

### ISSUE THREAD

**Issue by brewerj** [w3c/wai-intro-accessibility#20]
Overall I think that this introduction is an improvement over previous resources, and I appreciate the work on it. 

Compared to other sub-topics on the page though, I feel that the "Making the Web Accessible" subtopic receives proportionally less attention than it should, are that there are a few additional things that could be highlighted here to address this concern. For instance, the fact that the accessibility technical work has participation from different stakeholder groups; or that it happens within different groups and task forces within which one can participate to help build an accessible Web; or by giving a few more examples that only the example of WCAG, such as mentioning ATAG, or ARIA, or development of improved conformance accessibility testing. In the links provided, you could link over to one or two technical standards (perhaps WCAG and ARIA), and then link to the accessibility standards overview (fyi, currently a 404) for the rest. 

I don't feel strongly about these specific changes; but I do feel strongly that there should be at least a slightly increased emphasis of the extent and/or types of technical work, and some indication that this is something that there are ways to join on in and contribute to. Volume-wise, even adding one or two sentences, and another link or two under this topic, would IMO improve the balance of topics, and help this fit better in the context of a W3C WAI resource. Thanks...

**Comment by shawna-slh** [comment]
Draft resolution https://github.com/w3c/wai-intro-accessibility/pull/22/files?short_path=d680e8a#diff-d680e8a854a7cbad6d490c445cba2eba

**Comment by bakkenb** [comment]
+1 for draft. I like how the additions were implemented.

**Comment by sharronrush** [comment]
+1 to drafted solution, nicely done

**Comment by shawna-slh** [comment]
Proposed resolution implemented  -- and noted for EOWG review at https://www.w3.org/WAI/EO/wiki/Accessibility_Intros_Notes#Changes_since_18_January

I'm closing this for now -- Feel free to re-open if you want more consideration on it.

**Comment by brewerj** [comment]
+1 to the resolution as well. I think that this gracefully addresses my concern. I realized after filing the Github comment last night that another way to express my concern was that one of the most common misunderstandings that we hear about web accessibility work in general and WAI work specifically is the misunderstanding that it is all and only about WCAG. This particular "fix" avoids reinforcing that misunderstanding by right away putting a few other technical aspects in front of your reader, rather than counting on readers following links to yet another resource, which relatively few people are actually likely to do. Thanks.
